README.md

```diff
@@ -41,29 +41,62 @@ This plugin contains the following features:
 
     ```
-    - /azuredevops boards create
+    - /azuredevops boards create [title] [description]
     ```
     On successful creation of a work item, you will get a message from the bot with the details of the newly created work item.
 
-- Add subscriptions: A user can create subscriptions for a linked project to get messages in a selected channel for selected events on work items like Create, Update and Delete.
+- Add subscriptions: A user can create subscriptions for a linked project to get notifications in a selected channel for selected events on work items and pull requests.
 To add a new subscription for a linked project click on the project title under "Linked Projects" in RHS then click on the "Add new subscription" button in the subscription view. Users can also create subscriptions using the slash command below.
+    - For creating Boards subscriptions
 
     ```
-    - /azuredevops boards subscribe
+    - /azuredevops boards subscription add
     ```
 
-- View/List subscriptions: A user can view the list of subscriptions for a project by going on the subscriptions list page after clicking on the project title under "Linked Projects". Users can also view the list of all subscriptions for a channel by using the below slash command in the channel.
+    - For creating Repos subscriptions
 
     ```
-    - /azuredevops boards subscriptions
+    - /azuredevops repos subscription add
     ```
 
-- Delete subscriptions: A user can delete subscriptions for a project from RHS by going on the subscriptions list page after clicking on the project title under "Linked Projects". Users can also delete a subscription for a project by using the slash command below.
+    **Note:** Mattermost users who are project admins or team admins on the linked Azure DevOps project can only create/delete a subscription.
+
+- View/List subscriptions: A user can view the list of subscriptions for a project by going to the subscriptions list page after clicking on the project title under "Linked Projects". Users can also view the list of all subscriptions for a channel by using the below slash command in the channel.
+
+    - For listing Boards subscriptions
+
+    ```
+    - /azuredevops boards subscription list anyone all_channels
+    ```
+
+    - For listing Repos subscriptions
+
+    ```
+    - /azuredevops repos subscription list anyone all_channels 
+    ```
+
+    Supported filters on the above slash command:
+    - CreatedBy: `me`(show all subscriptions created by the current Mattermost user), `anyone`(show all subscriptions created by any Mattermost user)
+    - Show for all channels: when the filter `all_channels` is passed in the slash command then subscriptions for all channels are listed, you can leave this to list the subscriptions of the current channel only.
+
+    **Note:** Mattermost users who are project admins or team admins on the linked Azure DevOps project can only view/list subscriptions that exist in a channel where he/she is not a member.
```

Comment:
```suggestion
    **Note:** Mattermost users who are project admins or team admins on the linked Azure DevOps project can only view/list subscriptions that exist in a channel where he/she is not a member.
```
I think our motive here is `only admins can view/list subscriptions` but its sounding like `admins can only view/list subscription`

Reply:
yes but we need to read the entire sentence to get the complete meaning `admins can only view/list subscription in a channel where he is not a member`

Reply:
both the interpretation are same here